net.sf.eclipsecs.checkstyle/pom.xml

```diff
@@ -11,7 +11,7 @@
     <name>Checkstyle Core Library Plugin</name>
     
     <properties>
-        <checkstyle.version>8.35</checkstyle.version>
+        <checkstyle.version>8.36-SNAPSHOT</checkstyle.version>
```

Comment:
This will fail. It powers the `sourcepath` of this line: https://github.com/checkstyle/eclipse-cs/blob/master/net.sf.eclipsecs.checkstyle/.classpath#L3 So it needs to remain 8.35.